Connect4.py

```diff
@@ -1,7 +1,92 @@
 GRID_WIDTH = 7
 GRID_HEIGHT = 6
 
-grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+class Board:
+
+    def __init__(self):
+        self.grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+
+    def draw(self):
+        print("\n  0    1    2    3    4    5    6")
+        for row in self.grid:
+            print(row)
+
+    def drop(self, col):
+        col = int(col)
+        for i in range(5, -1, -1):
```

Comment:
5 is a magic number (GRID_HEIGHT - 1?) Use a constant to help the reader understand what's happening.